nerfstudio/viewer_beta/viewer_state.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2022 the Regents of the University of California, Nerfstudio Team and contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Manage the state of the viewer """
+from __future__ import annotations
+
+import threading
+from pathlib import Path
+from typing import TYPE_CHECKING, Literal, Optional
+
+import numpy as np
+import torch
+import torchvision
+import viser
+import viser.transforms as vtf
+
+from nerfstudio.configs import base_config as cfg
+from nerfstudio.data.datasets.base_dataset import InputDataset
+from nerfstudio.models.base_model import Model
+from nerfstudio.pipelines.base_pipeline import Pipeline
+from nerfstudio.utils.decorators import check_main_thread, decorate_all
+from nerfstudio.utils.writer import GLOBAL_BUFFER, EventName
+from nerfstudio.viewer.server import viewer_utils
+from nerfstudio.viewer_beta.render_state_machine import RenderAction, RenderStateMachine
+from nerfstudio.viewer_beta.utils import CameraState
+
+if TYPE_CHECKING:
+    from nerfstudio.engine.trainer import Trainer
+
+VISER_NERFSTUDIO_SCALE_RATIO: int = 10
```

Comment:
Hi, do you remember why you scaled up here? This creates weird distortions and makes the camera poses inconsistent between `COLMAP` and `NeRFStudio` visualizations.